packages/superset-ui-preset-chart-xy/src/utils/createTickComponent.tsx

```diff
@@ -1,13 +1,17 @@
 /* eslint-disable no-magic-numbers */
 
-import React from 'react';
-import PropTypes from 'prop-types';
+import React, { CSSProperties } from 'react';
 
 export default function createTickComponent({
   labellingStrategy,
   orientation = 'bottom',
   rotation = 40,
```

Comment:
should this be a constant? there are at least 3 files that set a default, and some don't match so it's tricky to figure out what the actual default will be.